src/implementations/twig/components/ordered-list/ordered-list.html.twig

```diff
@@ -48,21 +54,28 @@
   {% if _items is not empty %}
     <ol class="{{ _css_class }}" {{ _extra_attributes|raw }}>
       {% for _item in _items %}
-        {% if _item is not empty %}
-          <li class="ecl-ordered-list__item">
-            {{- _item.label -}}
-            {% if _item.nested is not empty and _item.nested is iterable %}
-              {{- list.ordered_list(_item.nested) -}}
-            {% endif -%}
-          </li>
+        {% set _item_class = 'ecl-ordered-list__item' %}
+        
+        {% if not nested and loop.first %}
+          {% set _item_class = _item_class ~ ' ecl-ordered-list__item--first' %}
```

Comment:
The thing is that this will also trigger the first and last items of the nested lists, but ok, still better to override a css rule than requesting additional classes..